docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,62 @@
+<!--[metadata]>
```

Comment:
That is a fair point. I don't think user expectation is that the data-plane is encrypted by default here. What we're talking about is the node-to-node communication, but it's still worth it to introduce the `--encrypted-data-plane` or whatever the flag is called.